### DISCUSSION TITLE
Parcel distribution takes huge time for the last node.

### DISCUSSION THREAD

**Discussion by tejaswinni51** [anacrolix/torrent#950] [Q&A]
Parcel distribution takes 25 mins for the first 98 nodes and 25 mins for the last node. The download bandwidth was reduced significantly from 300-400 MBs to 6-7 MBs. I'm using the [v1.56.0] tagged version.

**Comment by anacrolix** [comment]
Is there any chance you could try to bisect the issue? That is, find the commit that causes it? 

There were a few minor changes to requesting that I wouldn't think would cause a slowdown, but also BitTorrent V2 support which is a big change.

**Comment by anacrolix** [comment]
Could this be related https://github.com/anacrolix/torrent/issues/934? Or this https://github.com/anacrolix/torrent/discussions/953?
